File: src/states/states.ts
// Load states and capitals


import stateLGAs from "./states_and_lga";


export const states = Object.keys(stateLGAs);

export const LGAs: string[] = [];

Object.keys(stateLGAs).forEach((state) => {
    LGAs.push(...stateLGAs[state]);
});

export const getLGA = (state: string) => {
    return stateLGAs[state];
}

File: src/states/states.py
import json


with open('states.json') as f:
    states = json.load(f)

states_and_capitals = {}

for state in states:
    states_and_capitals[state['name']] = state['lgas']


with open('states_and_capitals.json', 'w') as f:
    json.dump(states_and_capitals, f, indent=4)
